x-pack/plugins/cloud_security_posture/common/constants.ts

```diff
@@ -19,8 +19,7 @@ export const FINDINGS_INDEX_DEFAULT_NS = 'logs-cloud_security_posture.findings-d
 
 export const LATEST_FINDINGS_INDEX_TEMPLATE_NAME = 'logs-cloud_security_posture.findings_latest';
 export const LATEST_FINDINGS_INDEX_PATTERN = 'logs-cloud_security_posture.findings_latest-*';
-export const LATEST_FINDINGS_INDEX_DEFAULT_NS =
-  'logs-cloud_security_posture.findings_latest-default';
+export const LATEST_FINDINGS_INDEX_DEFAULT_NS = 'cloud_security_posture.findings_latest-default';
 
```

Comment:
sine the `esArchives` failed to index documents to data stream, for make it work I removed the `logs-` preffix.